includes/class-wc-stripe-feature-flags.php

```diff
@@ -14,7 +14,8 @@ class WC_Stripe_Feature_Flags {
 	 * @return bool
 	 */
 	public static function is_stripe_ece_enabled() {
-		return 'yes' === get_option( self::ECE_FEATURE_FLAG_NAME, 'no' );
+		// @todo revert this
```

Comment:
Just adding a comment here to make sure we remember to revert this 😅

Reply:
Done! Reverted it